Prevent build from generating docs for both BN_print and bn_print.

http://trac.macports.org/ticket/31322

--- doc/crypto/bn_internal.pod.orig	2009-10-28 14:51:56.000000000 +0100
+++ doc/crypto/bn_internal.pod	2011-09-19 13:23:54.000000000 +0200
@@ -8,7 +8,7 @@
 bn_mul_low_normal, bn_mul_recursive, bn_mul_part_recursive,
 bn_mul_low_recursive, bn_mul_high, bn_sqr_normal, bn_sqr_recursive,
 bn_expand, bn_wexpand, bn_expand2, bn_fix_top, bn_check_top,
-bn_print, bn_dump, bn_set_max, bn_set_high, bn_set_low - BIGNUM
+bn_dump, bn_set_max, bn_set_high, bn_set_low - BIGNUM
 library internal functions
 
 =head1 SYNOPSIS
@@ -57,7 +57,6 @@
  void bn_fix_top(BIGNUM *a);
 
  void bn_check_top(BIGNUM *a);
- void bn_print(BIGNUM *a);
  void bn_dump(BN_ULONG *d, int n);
  void bn_set_max(BIGNUM *a);
  void bn_set_high(BIGNUM *r, BIGNUM *a, int n);
@@ -221,14 +220,14 @@
 bn_check_top() verifies that C<((a)-E<gt>top E<gt>= 0 && (a)-E<gt>top
 E<lt>= (a)-E<gt>dmax)>.  A violation will cause the program to abort.
 
-bn_print() prints B<a> to stderr. bn_dump() prints B<n> words at B<d>
+bn_dump() prints B<n> words at B<d>
 (in reverse order, i.e. most significant word first) to stderr.
 
 bn_set_max() makes B<a> a static number with a B<dmax> of its current size.
 This is used by bn_set_low() and bn_set_high() to make B<r> a read-only
 B<BIGNUM> that contains the B<n> low or high words of B<a>.
 
-If B<BN_DEBUG> is not defined, bn_check_top(), bn_print(), bn_dump()
+If B<BN_DEBUG> is not defined, bn_check_top(), bn_dump()
 and bn_set_max() are defined as empty macros.
 
 =head1 SEE ALSO
